superset/common/query_context.py

```diff
@@ -260,14 +258,8 @@ def get_df_payload(  # pylint: disable=too-many-locals,too-many-statements
             if is_loaded and cache_key and cache and status != utils.QueryStatus.FAILED:
                 try:
                     cache_value = dict(dttm=cached_dttm, df=df, query=query)
-                    cache_binary = pkl.dumps(cache_value, protocol=pkl.HIGHEST_PROTOCOL)
-
-                    logger.info(
```

Comment:
I annexed this as we no longer a priori know the size of the payload and it seems somewhat like an unnecessary comment.